include/MainApplication.h

```diff
@@ -37,6 +37,13 @@
 
 namespace lmms::gui
 {
+// for Qt 6, there's a signature change in nativeEventFilter, to clean once 
+// Qt 5 support is dropped
+#if (QT_VERSION >= QT_VERSION_CHECK(6,0,0))
+using filterResult = quintptr;
```

Comment:
A few things:
* Our coding conventions require type names to start with an uppercase letter (i.e. `FilterResult`).
* The appropriate type here is `qintptr`, rather than `quintptr`.
* I think the comment is redundant - it doesn't say anything that isn't clear from the code.